src/main.ts

```diff
@@ -3,7 +3,8 @@ import {
   requireEvent,
   requireFiles,
   requirePr,
-  requirePullNumber
+  requirePullNumber,
+  require_cc0
```

Comment:
```suggestion
  require_cc0,
  require_created_at
```
I think you forgot to import one of the new requires.